index.js

```diff
@@ -51,46 +51,54 @@ exports.X509 = {
  */
 exports.SSH_TUNNEL_IDENTITY_FILE = {
   name: 'SSH tunnel connection using an identity file',
-  hostname: 'standalone.compass-test-1.mongodb.parts',
+  hostname: '10.0.0.131',
   port: 27000,
-  ns: 'fanclub',
   mongodb_username: 'integrations',
   mongodb_password: process.env.MONGODB_PASSWORD_INTEGRATIONS,
-  ssh_tunnel_hostname: process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_HOST,
-  ssh_tunnel_user: process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_USER,
-  ssh_tunnel_identity_file: [ process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE ]
+  mongodb_database_name: 'admin',
+  ssh_tunnel: 'IDENTITY_FILE',
+  ssh_tunnel_hostname: 'jumpbox.compass-test-ssh.mongodb.parts',
+  ssh_tunnel_port: 22,
+  ssh_tunnel_bind_to_local_port: 29799,
+  ssh_tunnel_username: 'ec2-user',
+  ssh_tunnel_identity_file: [ process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE ],
+  ssh_tunnel_passphrase: ''
 };
 
 /**
  * For SSH tunnel connections with an identity file and passphrase.
+ *
+ * @todo (thomasr) currently not used.
  */
-exports.SSH_TUNNEL_IDENTITY_FILE_WITH_PASSPHRASE = {
-  name: 'SSH tunnel connection using an identity file with passphrase',
-  hostname: 'standalone.compass-test-1.mongodb.parts',
-  port: 27000,
-  ns: 'fanclub',
-  mongodb_username: 'integrations',
-  mongodb_password: process.env.MONGODB_PASSWORD_INTEGRATIONS,
-  ssh_tunnel_hostname: process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_WITH_PASSPHRASE_HOST,
-  ssh_tunnel_user: process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_WITH_PASSPHRASE_USER,
-  ssh_tunnel_identity_file: [ process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_WITH_PASSPHRASE ],
-  ssh_tunnel_passphrase: process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_PASSPHRASE
-};
+// exports.SSH_TUNNEL_IDENTITY_FILE_WITH_PASSPHRASE = {
+//   name: 'SSH tunnel connection using an identity file with passphrase',
+//   hostname: 'standalone.compass-test-1.mongodb.parts',
+//   port: 27000,
+//   ns: 'fanclub',
+//   mongodb_username: 'integrations',
+//   mongodb_password: process.env.MONGODB_PASSWORD_INTEGRATIONS,
+//   ssh_tunnel_hostname: process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_WITH_PASSPHRASE_HOST,
+//   ssh_tunnel_user: process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_WITH_PASSPHRASE_USER,
+//   ssh_tunnel_identity_file: [ process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_WITH_PASSPHRASE ],
+//   ssh_tunnel_passphrase: process.env.MONGODB_SSH_TUNNEL_IDENTITY_FILE_PASSPHRASE
+// };
 
 /**
  * For SSH tunnel connections with username/password.
+ *
+ * @todo (thomasr) currently not used.
```

Comment:
@rueckstiess I'd personally drop this scenario entirely rather than leaving it commented out.

* I don't see why [authorization](https://docs.mongodb.com/master/core/authorization/) and SSH should be tested in isolation, both together is a common and expected production scenario.
* i.e. we should just test `SSH_TUNNEL_IDENTITY_FILE` and `SSH_TUNNEL_IDENTITY_FILE_WITH_PASSPHRASE` (plus they all have a username/password combo anyway).

Remainder looks fine to be handled in COMPASS-565 👍